README.md

```diff
@@ -137,45 +158,31 @@ $ make CFG_TEE_CORE_LOG_LEVEL=4
 ```
 
 ---
-### 4.2 Foundation Models
-
-See section [6. repo manifests]((#6-repo-manifests).
-
----
-### 4.3 Juno
-Juno has been supported in OP-TEE since mid October 2014.
+### 4.3 ARM Juno board
++ The script `setup_juno_optee.sh` script provides a coherent set of components
+(OP-TEE client, driver, OS, Linux kernel version 3-16.0-rc5)
 
-#### WARNING:
-
-+ The ```setup_juno_optee.sh``` script provides a coherent set of components (OP-TEE client/driver/os,
-Linux kernel version 3-16.0-rc5)
-
-+ Further release will align the ARM Juno setup with other OP-TEE supported platforms:
++ Futures releases will align the Juno setup with other OP-TEE supported
+  platforms:
 
 	+ Linux kernel version alignment (3.18-rc1) with QEMU/FVP (DMA_BUF API change).
 	+ Will need arch/arm/Kconfig patch(es) (i.e DMA_SHARED_BUFFER etc...).
 
-+ Temporary patch files required for linux kernel and juno dtb definition:
++ Temporary patch files required for linux kernel and Juno DTB definition (found
+  in the `./scripts` folder)
 
-	+ config.linux-linaro-tracking.a226b22057c22b433caafc58eeae6e9b13ac6c8d.patch
-	+ juno.dts.linux-linaro-tracking.a226b22057c22b433caafc58eeae6e9b13ac6c8d.patch
+	+ `config.linux-linaro-tracking.a226b22057c22b433caafc58eeae6e9b13ac6c8d.patch`
+	+ `juno.dts.linux-linaro-tracking.a226b22057c22b433caafc58eeae6e9b13ac6c8d.patch`
 
-#### 4.3.1 Prerequisites
-+ The following packages must be installed:
+#### 4.3.1 Prerequisites for Juno board
++ Download pre-built binaries (please see "Known issues" below)
 
-```
-$ sudo apt-get install zlib1g-dev libglib2.0-dev libpixman-1-dev libfdt-dev \
-		       libc6:i386 libstdc++6:i386 libz1:i386 cscope netcat
-```
-
-+ Download ARM Juno pre-built binaries:
+	+ Juno boards pre-built binary `bl30.bin` (SCP runtime)
+	+ Juno boards pre-built binary `bl33.bin` (UEFI)
+	+ Download at &rarr; https://releases.linaro.org/members/arm/platforms/latest
```

Comment:
Ok, I'll change this sections slightly. I'll make sure it's clear that the current setup isn't working and we are working on solving that issue.